storage/google_cloud/google_cloud_utils.py

```diff
@@ -84,9 +86,31 @@ def upload_file_to_blob(bucket_credentials_file_path, target_blob_url, f):
     :type target_blob_url: str
     :param f: File to upload, opened in binary mode.
     :type f: file-like
+    :param max_retries: maximum number of times to retry uploading the file.
+    :type max_retries: int
+    :param blob_chunk_size: the size of a chunk of data whenever iterating (in MiB).
```

Comment:
I don't understand what you mean by "iterating" here.
Struggling to think of something better, but I think we can just say this:
```suggestion
    :param blob_chunk_size: The chunk size to use for resumable uploads, in MiB
```
You need to know about resumable uploads and chunk sizes anyway to understand how to set this, so I think that's ok?